tests/run.rs

```diff
@@ -0,0 +1,48 @@
+extern crate livesplit_core;
+
+mod run {
+    use livesplit_core::{Run, Segment};
+    use livesplit_core::run::Editor;
+
+    #[test]
+    fn add_comparison() {
+        let mut run = Run::new();
+        let c = run.add_custom_comparison("My Comparison").ok();
+        assert_eq!(c, Some(()));
```

Comment:
instead of using .ok() to convert to Option you can just assert against Ok(()), same for the other tests